examples/droplet/main.tf

```
@@ -23,6 +32,7 @@ resource "digitalocean_droplet" "mywebserver" {
     ]
 
     connection {
+      host     = self.ipv4_address
       type     = "ssh"
       private_key = "${file("~/.ssh/id_rsa")}"
```

While this isn't part of your change, I think this is a good opportunity to clear that deprecation warning

```suggestion
      private_key = file("~/.ssh/id_rsa")
```